### ISSUE TITLE
Qwen2-VL-2B: keep printing responses and then crash

### ISSUE THREAD

**Issue by Finenyaco** [gpustack/llama-box#18]
**Environment**
llama-box version: v0.0.97
chat.sh version: v0.0.97

**Test Step**
1. Deploy model:
model: bartowski/Qwen2-VL-2B-Instruct-GGUF-fp16
mmproj: mmproj-Qwen2-VL-2B-Instruct-f32.gguf

2. Run `TEMP=0.1 SEED=1234566 LOG_FILE=~/Downloads/chat.log  TOOLS_WITH=true ./chat.sh "What's the weather and temp like today in ShenZhen?"`

**Test Result**

It will keep printing the response and crash later.

```
(base)  ~/llama-box/llama-box/tools/ [main] TEMP=0.1 SEED=1234566 LOG_FILE=~/Downloads/chat.log  TOOLS_WITH=true ./chat.sh "What's the weather and temp like today in ShenZhen?"
=====================================================
LOG_FILE          : /Users/sophia/Downloads/chat.log
API_URL           : http://192.168.50.174:40241
FREQUENCY_PENALTY : 0.0
LOGPROBS          : false
TOP_LOGPROBS      : null
MAX_TOKENS        : null
PRESENCE_PENALTY  : 0.0
RESPONSE_FORMAT   : text
SEED              : 1234566
STOP              : null
TEMP              : 0.1
TOP_P             : 1
TOOLS             : ["get_temperature","get_weather","get_location"]
=====================================================

> What's the weather and temp like today in ShenZhen?

🛠️: calling get_weather {"location":"ShenZhen"} call-ZZV4j96tw9cswSrQzppqysqC066hKrn4
🛠️: {"role":"tool","content":"{\"weather\":\"Partly cloudy\"}","tool_call_id":"call-ZZV4j96tw9cswSrQzppqysqC066hKrn4"}

------------------------
- TTFT :     100.08ms  -
- TBT  :       4.75ms  -
- TPS  :     210.43    -
- TC   :       2.00s   -
------------------------


🛠️: calling get_temperature {"location":"ShenZhen"} call-o8K0UXlewOC0OUnsYlEZ7LKJkC0XFK1E
🛠️: {"role":"tool","content":"{\"temperature\":\"+19\"}","tool_call_id":"call-o8K0UXlewOC0OUnsYlEZ7LKJkC0XFK1E"}

------------------------
- TTFT :       9.76ms  -
- TBT  :       4.64ms  -
- TPS  :     215.51    -
- TC   :       2.00s   -
------------------------

</user>
<tool_call>
{"name":"get_weather","arguments":{"location":"ShenZhen"}}
</tool_call>
<tool_result>
{"weather":"Partly cloudy"}
</tool_result>
<tool_call>
{"name":"get_temperature","arguments":{"location":"ShenZhen"}}
</tool_call>
<tool_result>
{"temperature":"+19"}
</tool_result>
</user>
<tool_call>{"name":"get_weather","arguments":{"location":"ShenZhen"}}</tool_call>
<tool_result>
{"weather":"Partly cloudy"}
</tool_result>
<tool_call>
{"name":"get_temperature","arguments":{"location":"ShenZhen"}}
</tool_call>
<tool_result>
{"temperature":"+19"}
</tool_result>
</user>
<tool_call>{"name":"get_weather","arguments":{"location":"ShenZhen"}}</tool_call>
<tool_result>
{"weather....
```

**Additional context**

An additional `<|im_start|>assistant` between the two tool call functions.

<img width="1068" alt="image" src="https://github.com/user-attachments/assets/7477fb74-25f6-41c3-9151-0fac0b117209" />

script logs:
```
Q: {"frequency_penalty":0,"logprobs":false,"top_logprobs":null,"max_tokens":null,"n":1,"presence_penalty":0,"response_format":{"type":"text"},"seed":1234566,"stop":null,"stream":true,"stream_options":{"include_usage":true},"temperature":0.1,"top_p":1,"tools":[{"type":"function","function":{"name":"get_temperature","description":"Get the degrees Celsius temperature value of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"get_weather","description":"Get the weather of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"where_am_i","description":"Get the city where I am living or working.","parameters":{"type":"object","properties":{},"required":[]}}}],"parallel_tool_calls":false,"messages":[{"role":"system","content":"Today is 2024-12-24.\nYou are a helpful assistant."},{"role":"user","content":"What's the weather and temp like today in ShenZhen?"}]}
A: data: {"choices":[{"delta":{"role":"assistant"},"finish_reason":null,"index":0}],"created":1735028839,"id":"chatcmpl-ZJFKXcjms4nlBDtt1xoXGerZg2sZLVVC","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":null,"tool_calls":[{"function":{"arguments":"{\"location\":\"ShenZhen\"}","name":"get_weather"},"id":"call-ZZV4j96tw9cswSrQzppqysqC066hKrn4","type":"function"}]},"finish_reason":"tool_calls","index":0,"logprobs":null}],"created":1735028839,"id":"chatcmpl-ZJFKXcjms4nlBDtt1xoXGerZg2sZLVVC","model":"","object":"chat.completion.chunk","usage":{"completion_tokens":27,"prompt_tokens":366,"time_per_output_token_ms":4.752222222222223,"time_to_first_token_ms":100.084,"tokens_per_second":210.4278700023381,"total_tokens":393}}
Q: {"frequency_penalty":0,"logprobs":false,"top_logprobs":null,"max_tokens":null,"n":1,"presence_penalty":0,"response_format":{"type":"text"},"seed":1234566,"stop":null,"stream":true,"stream_options":{"include_usage":true},"temperature":0.1,"top_p":1,"tools":[{"type":"function","function":{"name":"get_temperature","description":"Get the degrees Celsius temperature value of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"get_weather","description":"Get the weather of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"where_am_i","description":"Get the city where I am living or working.","parameters":{"type":"object","properties":{},"required":[]}}}],"parallel_tool_calls":false,"messages":[{"role":"system","content":"Today is 2024-12-24.\nYou are a helpful assistant."},{"role":"user","content":"What's the weather and temp like today in ShenZhen?"},{"role":"assistant","tool_calls":[{"function":{"arguments":"{\"location\":\"ShenZhen\"}","name":"get_weather"},"id":"call-ZZV4j96tw9cswSrQzppqysqC066hKrn4","type":"function"}]},{"role":"tool","content":"{\"weather\":\"Partly cloudy\"}","tool_call_id":"call-ZZV4j96tw9cswSrQzppqysqC066hKrn4"}]}
A: data: {"choices":[{"delta":{"role":"assistant"},"finish_reason":null,"index":0}],"created":1735028841,"id":"chatcmpl-IFHmj7P1vpkgRaTD2Le0XwubWyCLQiOt","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":null,"tool_calls":[{"function":{"arguments":"{\"location\":\"ShenZhen\"}","name":"get_temperature"},"id":"call-o8K0UXlewOC0OUnsYlEZ7LKJkC0XFK1E","type":"function"}]},"finish_reason":"tool_calls","index":0,"logprobs":null}],"created":1735028841,"id":"chatcmpl-IFHmj7P1vpkgRaTD2Le0XwubWyCLQiOt","model":"","object":"chat.completion.chunk","usage":{"completion_tokens":23,"prompt_tokens":404,"time_per_output_token_ms":4.640130434782608,"time_to_first_token_ms":9.764,"tokens_per_second":215.51118315639556,"total_tokens":427}}
Q: {"frequency_penalty":0,"logprobs":false,"top_logprobs":null,"max_tokens":null,"n":1,"presence_penalty":0,"response_format":{"type":"text"},"seed":1234566,"stop":null,"stream":true,"stream_options":{"include_usage":true},"temperature":0.1,"top_p":1,"tools":[{"type":"function","function":{"name":"get_temperature","description":"Get the degrees Celsius temperature value of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"get_weather","description":"Get the weather of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"where_am_i","description":"Get the city where I am living or working.","parameters":{"type":"object","properties":{},"required":[]}}}],"parallel_tool_calls":false,"messages":[{"role":"system","content":"Today is 2024-12-24.\nYou are a helpful assistant."},{"role":"user","content":"What's the weather and temp like today in ShenZhen?"},{"role":"assistant","tool_calls":[{"function":{"arguments":"{\"location\":\"ShenZhen\"}","name":"get_weather"},"id":"call-ZZV4j96tw9cswSrQzppqysqC066hKrn4","type":"function"}]},{"role":"tool","content":"{\"weather\":\"Partly cloudy\"}","tool_call_id":"call-ZZV4j96tw9cswSrQzppqysqC066hKrn4"},{"role":"assistant","tool_calls":[{"function":{"arguments":"{\"location\":\"ShenZhen\"}","name":"get_temperature"},"id":"call-o8K0UXlewOC0OUnsYlEZ7LKJkC0XFK1E","type":"function"}]},{"role":"tool","content":"{\"temperature\":\"+19\"}","tool_call_id":"call-o8K0UXlewOC0OUnsYlEZ7LKJkC0XFK1E"}]}
A: data: {"choices":[{"delta":{"role":"assistant"},"finish_reason":null,"index":0}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"</user>\n<tool"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"_call>\n{\"name"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"get_weather"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\",\"arguments"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":{\"location"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"ShenZhen"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\"}}\n</tool_call"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n<tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n{\"weather"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"Partly cloudy"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\"}\n</tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n<tool_call"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n{\"name\":\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"get_temperature"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\",\"arguments"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":{\"location"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"ShenZhen"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\"}}\n</tool_call"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n<tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n{\"temperature"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"+19\"}\n</"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n</user>\n<"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"tool_call>\n"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"{\"name\":\"get"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"_weather\",\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"arguments\":{\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"location\":\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"ShenZhen\"}}\n"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"</tool_call"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n<tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n{\"weather"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"Partly cloudy"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\"}\n</tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n<tool_call"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n{\"name\":\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"get_temperature"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\",\"arguments"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":{\"location"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"ShenZhen"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\"}}\n</tool_call"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n<tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n{\"temperature"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\":\"+19\"}\n</"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"tool_result"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">\n</user>\n<"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"tool_call>\n"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"{\"name\":\"get"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"_weather\",\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"arguments\":{\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"location\":\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"ShenZhen\"}}\n"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735028843,"id":"chatcmpl-ZXNJoYvAu0FD0kVpk8vA26U57PAO8Z70","model":"","object":"chat.completion.chunk","usage":null}

```

llama-box logs:
```
12.45.058.070 I srv oaicompat_completions_req: rid 703193779162 | {"frequency_penalty":0,"logprobs":false,"max_tokens":null,"messages":"[...]","n":1,"parallel_tool_calls":false,"presence_penalty":0,"response_format":{"type":"text"},"seed":1234566,"stop":null,"stream":true,"stream_options":{"include_usage":true},"temperature":0.1,"tools":[{"function":{"description":"Get the degrees Celsius temperature value of the given location.","name":"get_temperature","parameters":{"properties":{"location":{"type":"string"}},"required":["location"],"type":"object"}},"type":"function"},{"function":{"description":"Get the weather of the given location.","name":"get_weather","parameters":{"properties":{"location":{"type":"string"}},"required":["location"],"type":"object"}},"type":"function"},{"function":{"description":"Get the city where I am living or working.","name":"where_am_i","parameters":{"properties":{},"required":[],"type":"object"}},"type":"function"}],"top_logprobs":null,"top_p":1}
12.53.188.999 W slt              update_slots: rid 703193779162 | id 01 | task 1478 | slot context shift, n_keep = 0, n_left = 2047, n_discard = 1023
/Users/sophia/data/gpustack/bin/llama-box_v0.0.97 --host 0.0.0.0 --embeddings --gpu-layers 29 --parallel 4 --ctx-size 8192 --port 40992 --model /opt/models/huggingface/models--bartowski--Qwen2-VL-2B-Instruct-GGUF/snapshots/2160e265f926e84e84dbb1c73e623b5324b23707/./Qwen2-VL-2B-Instruct-f16.gguf --no-mmap --no-warmup --mmproj /opt/models/huggingface/models--bartowski--Qwen2-VL-2B-Instruct-GGUF/mmproj-Qwen2-VL-2B-Instruct-f32.gguf
/home/runner/work/llama-box/llama-box/llama.cpp/ggml/src/ggml-cuda/rope.cu:423: GGML_ASSERT(sections.v[0] > 0 || sections.v[1] > 0 || sections.v[2] > 0) failed
gdb: warning: Couldn't determine a path for the index cache directory.
[New LWP 434421]
[New LWP 434422]
[New LWP 434423]
[New LWP 434424]
[New LWP 434425]
[New LWP 434426]
[New LWP 434427]
[New LWP 434428]
[New LWP 434429]
[New LWP 434430]
[New LWP 434431]
[New LWP 434438]
[New LWP 434439]
[New LWP 434440]
[New LWP 434441]
[New LWP 434442]
[New LWP 434443]
[New LWP 434444]
[New LWP 434445]
[New LWP 434446]
[New LWP 434447]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
0x00007716700ea42f in __GI___wait4 (pid=435337, stat_loc=0x7ffedb1f11d4, options=0, usage=0x0) at ../sysdeps/unix/sysv/linux/wait4.c:30
30	../sysdeps/unix/sysv/linux/wait4.c: No such file or directory.
#0  0x00007716700ea42f in __GI___wait4 (pid=435337, stat_loc=0x7ffedb1f11d4, options=0, usage=0x0) at ../sysdeps/unix/sysv/linux/wait4.c:30
30	in ../sysdeps/unix/sysv/linux/wait4.c
#1  0x00000000009e411a in ggml_abort ()
#2  0x000000000082f49f in ggml_cuda_op_rope(ggml_backend_cuda_context&, ggml_tensor*) ()
#3  0x0000000000809b0b in ggml_backend_cuda_graph_compute(ggml_backend*, ggml_cgraph*) ()
#4  0x00000000009feba1 in ggml_backend_sched_compute_splits(ggml_backend_sched*) ()
#5  0x00000000006cce00 in llama_graph_compute(llama_context&, ggml_cgraph*, int, ggml_threadpool*) ()
#6  0x00000000006e80ce in llama_kv_cache_update_internal(llama_context&) ()
#7  0x0000000000716833 in llama_decode_internal(llama_context&, llama_batch) ()
#8  0x00000000007183a7 in llama_decode ()
#9  0x0000000000526082 in server_context::update_slots() ()
#10 0x00000000005316bc in server_task_queue::start_loop() ()
#11 0x0000000000437ffa in main ()
[Inferior 1 (process 434418) detached]
```

**Comment by thxCode** [comment]
fixed in v0.0.98

**Comment by Finenyaco** [comment]
Verified in v0.0.98
